### PR TITLE
Added Banano support

### PR DESCRIPTION
**Pull request by Charterino**

### CODE REVIEW

File: Nano.Net/Utils.cs [hunk 1]
```diff
@@ -144,29 +144,31 @@ private static string EncodedAddressChecksum(byte[] publicKey)
             return EncodeNanoBase32(final.Reverse().ToArray(), false);
         }
 
-        public static string AddressFromPublicKey(byte[] publicKey)
+        public static string AddressFromPublicKey(byte[] publicKey, string prefix)
```
Comment (miguel1117) [inline Nano.Net/Utils.cs]: All methods that have been modified to work with different prefixes should still work without explicitly choosing a prefix, using the nano_ prefix by default, in order to preserve backwards compatibility and to simplify usage

File: Nano.Net/RpcClient.cs [hunk 2]
```diff
@@ -88,8 +88,17 @@ public async Task<AccountInfoResponse> AccountInfoAsync(string address, bool rep
         /// <remarks>
         /// WARNING: This command is usually disabled on public nodes. You need to use your own node.
         /// </remarks>
-        public async Task<WorkGenerateResponse> WorkGenerateAsync(string hash)
+        public async Task<WorkGenerateResponse> WorkGenerateAsync(string hash, string difficulty = "")
```
Comment (miguel1117) [inline Nano.Net/RpcClient.cs]: The default difficulty argument should be null and not an empty string to maintain consistency across the library.

**Review by miguel1117 — CHANGES_REQUESTED** [review]

File: Nano.Net/Utils.cs (hunk 1 above)
Comment (Charterino) [reply Nano.Net/Utils.cs]: Please check 280e65b and 2d56fe4.

File: Nano.Net/RpcClient.cs (hunk 2 above)
Comment (Charterino) [reply Nano.Net/RpcClient.cs]: Please check 981ddd0.

**Pull request merged**